config/environments/production.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.configure do
   # Specify environment specific hostname and protocol
-  config.hostname = ENV["HOSTNAME"]
+  config.hostname = "#{ENV["HOSTNAME"]}/api"
```

Comment:
https://github.com/department-of-veterans-affairs/devops/pull/564

I think leaving it as HOSTNAME is fine. we should be getting that DNS soon, so leaving the name as inaccurate for a week seems worthwhile relative to the gain of changing the name then changing it back.